components/atoms/ActionButton.js

```diff
@@ -4,47 +4,38 @@ import Link from 'next/link'
 /**
  * Button component
  */
-export function ActionButton({
-  href,
-  className,
-  onClick,
-  type,
-  lang,
-  id,
-  text,
-  children,
-  ...rest
-}) {
+export function ActionButton(props) {
+  var { ...rest } = props
```

Comment:
Use `const` if possible, please. Otherwise, use `let` :)  Gotta break those old habits ;)

Reply:
Why am I like this